apps/portals/eliteportal/src/config/synapseConfigs/cohortbuilder.ts

```diff
@@ -120,7 +120,7 @@ export const filesViewQueryWrapperPlotNavProps: QueryWrapperPlotNavProps = {
   name: 'Data',
   enabledExternalAnalysisPlatforms: enabledAnalysisPlatforms,
   fileIdColumnName: 'id',
-  fileNameColumnName: 'name',
+  fileNameColumnName: 'fileName',
```

Comment:
Use String column that contains the actual file name (the "name" column is an Entity type column that contains the Synapse ID)